packages/manager/apps/dedicated/client/app/account/user/support-level/translations/Messages_fr_FR.json

```diff
@@ -1,24 +1,35 @@
-   "user_account_support_level_section_business_description": "Un service client 24/7 avec un temps de réponse optimisé adapté à des enjeux  majeurs.",
-   "user_account_support_level_section_enterprise_description": "Un service client 24/7 au travers d’une relation privilégiée avec une équipe dédiée pour vous accompagner à relever vos défis business.",
-   "user_account_support_level_section_title_annex_1": "Nos offres supports évoluent. Veuillez noter que par défaut tous nos clients qui ne souscrivent pas à une offre de support bénéficieront de l'offre standard.",
-   "user_account_support_level_section_title_annex_all_available": "Tout client OVHcloud est par défaut basculé vers notre niveau de support Standard s’il ne souscrit pas à une offre de support spécifique."
+  "user_account_support_level_section_title_what": "Quels sont les différents niveaux de support ?",
+  "user_account_support_level_section_title_subscribe": "Vous avez néanmoins le choix de garder votre niveau de support actuel, ou de souscrire à un niveau autre que celui recommandé.",
+  "user_account_support_level_section_title_description": "Pour répondre à chacune de vos spécificités, OVHcloud a mis en place 4 niveaux d’accompagnement<sup aria-label=\"(1)\">*</sup>. Par défaut, nos solutions sont livrées avec le niveau Standard. Selon les solutions auxquelles vous souscrivez, nous vous recommandons un niveau de support qui vous fera bénéficier au mieux d’un accompagnement de nos experts. Vous avez néanmoins le choix de garder votre niveau de support actuel, ou de souscrire à un niveau autre que celui recommandé.",
+  "user_account_support_level_section_title_description_standard_standard": "Vous êtes un Standard Partner. En tant que revendeur à valeur ajoutée et/ou intégrateur système, vous devez souscrire au support Premium. Ce niveau vous garantit un accès privilégié à nos équipes d'assistance, ainsi qu'un traitement prioritaire de vos demandes. À noter que cette obligation contractuelle ne concerne pas votre activité d’infogérance, de conseil ou de formation.",
+  "user_account_support_level_section_title_description_standard_premium": "Vous êtes un Standard Partner et vous avez souscrit au support Premium. Ce niveau vous garantit une meilleure qualité de service avec, notamment, un accès privilégié à nos équipes d’assistance ainsi qu’un traitement prioritaire de vos demandes.",
+  "user_account_support_level_section_title_description_standard_premium-accredited": "Vous êtes récemment devenu un Standard Partner. Vous ne pouvez donc plus souscrire au support Premium for Advanced Partner. Cependant, en tant que revendeur à valeur ajoutée et/ou intégrateur système, vous devez souscrire au support Premium. Ce niveau vous garantit un accès privilégié à nos équipes d'assistance, ainsi qu'un traitement prioritaire de vos demandes. À noter que cette obligation contractuelle ne concerne pas votre activité d’infogérance, de conseil ou de formation.",
```

Comment:
Some discussion about used wording is currently going on, so translations might get an update